packages/sdks/src/functions/get-content/index.ts

```diff
@@ -87,19 +95,24 @@ export const processContentResult = async (
   return content;
 };
 
-export async function getAllContent(options: GetContentOptions) {
+export async function fetchAllEntries(options: GetContentOptions) {
```

Comment:
i would suggest calling this `fetchEntries` since that makes it clear it's plural and removes `all` since we saw repeat confusion that people thought `all` means actually give me everything, vs just give me multiple. we could call it `fetchMultipleEntries` or `fetchEntriesList` but feels like `fetchEntries` is nicer